content/docs-v1/pages.md

```diff
@@ -101,45 +100,44 @@ In the following example publication outline, we’ve listed the files and direc
-
-```yaml
-title: Part One
-type: contents
-class: grid
-slug: .
-```
+A Quire publication can have sub-sections, created by nesting a group of one or more pages inside a sub-directory within the main `content` directory. It is recommended (though not required) to designate one of the pages in each sub-directory to be the section landing page. To do so, name the `.md` file for the desired landing page `index.md`. The `title` of your `index.md` file is not only what will be used in places like the header of that page, the table of contents, and the menu of your site, it will also be used to define the URL. For example, if the `title` value of your `index.md` page is `"Landing Page"` then the URL would appear as `my-project.com/landing-page` rather than `my-project.com/index.md`
 
-The `title` of your defined landing page is what will be used in the header of that page, the *Table of Contents*, and the menu of your site.
+## Create Publication Cover Page
 
-However, the filename of the sub-directory itself is also used in your publication; for the online navigation bar, and in the running page footers of the PDF version. In these two places, Quire takes the sub-directory filename and humanizes it, which means to change hyphens into spaces and capitalize with title case. So, the sub-directory `part-one` becomes “Part One”, or `sculpture-of-the-renaissance` becomes “Sculpture of the Renaissance.”
+The way to create a publication cover page is similar to creating section landing pages. Name the `.md` file for your cover `index.md` and include the `layout` value of `cover`. The cover is usually given a page `order` of `1`. You can also include an attribute of `image` on your cover page and a link to the file. For example, `figures/my-cover-image.jpg`. You may also want to exclude the cover from the menu and table of contents. Learn how to do that in the following section.
 
 ## Hide/Show Pages
 
-By default, every page you create will be included in all formats of your publication (online, PDF/print, and e-book). Every page will also automatically be listed in the publication’s menu and contents pages. However, this can be overridden by setting any of the following Page YAML attributes to `false`.
+By default, every page you create will be included in all formats of your publication (online, PDF/print, and e-book). This can be overridden by including an `outputs` attribute and excluding the undesired formats from the array (`epub`, `pdf`, `online`). For example, if you want your Copyright page to appear in the PDF and EPUB but not in the online version, you would exclude `online` from the YAML.
+
+```YAML
+outputs: [epub, pdf]
```

Comment:
Is it worth mentioning that this may also be written this way?

```yaml
outputs:
  - epub
  - pdf
```

Reply:
I think it would be best to pick one, and I actually think your latter suggestion is the better way to go as it's more in line with other YAML formatting that appears in the docs. The use of brackets is kind of an anomaly.

Reply:
Ah yes, good call.